.env.example

```
@@ -0,0 +1,13 @@
+MYSQLDB_ROOT_PASSWORD=rootpassword
+MYSQLDB_USER=user
+MYSQLDB_PASSWORD=password
+MYSQLDB_DATABASE=database
+MYSQL_LOCAL_PORT=3308
+MYSQL_DOCKER_PORT=3306
+
+JWT_EXPIRATION=3600000
+JWT_SECRET=secret
```

Why there are two passwords MYSQLDB_ROOT_PASSWORD and MYSQLDB_PASSWORD? 

```suggestion
MYSQLDB_ROOT_PASSWORD=
MYSQLDB_USER=
MYSQLDB_PASSWORD=
MYSQLDB_DATABASE=
MYSQL_LOCAL_PORT=
MYSQL_DOCKER_PORT=

JWT_EXPIRATION=
JWT_SECRET=
```